src/app/components/UserPasswordChange.js

```diff
@@ -1,81 +1,111 @@
-    return (
-      <FormattedMessage id="passwordChange.title" defaultMessage="Change password">
-        { label => (
-          <PageTitle prefix={label}>
-            <StyledPasswordChange>
-              { this.state.showConfirmDialog ?
-                <Card className="user-password-change__confirm-card">
-                  <CardHeader title={<FormattedMessage id="passwordChange.successTitle" defaultMessage="Password updated" />} />
-                  <CardContent>
-                    <FormattedMessage
-                      id="passwordChange.successMsg"
-                      defaultMessage="You're all set. Now you can log in with your new password."
-                    />
-                  </CardContent>
-                  <CardActions className="user-password-change__actions">
-                    <Button color="primary" onClick={handleSignIn}>
-                      <FormattedMessage id="passwordChange.signIn" defaultMessage="Got it" />
-                    </Button>
-                  </CardActions>
-                </Card> :
-                <Card className="user-password-change__card">
-                  <CardContent>
-                    <img alt="" src={stringHelper('LOGO_URL')} className="user-password-change__logo" />
+  const classes = useStyles();
 
-                    <span className="user-password-change__title">
-                      <FormattedMessage id="passwordChange.title" defaultMessage="Change password" />
-                    </span>
+  return (
+    <PageTitle>
+      <p className={classes.bestViewed}>
+        <FormattedHTMLMessage {...globalStrings.bestViewed} />
+      </p>
+      <ContentColumn center className="user-password-reset__component">
+        <StyledCard>
+          <FormattedGlobalMessage messageKey="appNameHuman">
+            {appNameHuman => (
+              <img
+                className={classes.logo}
+                alt={appNameHuman}
+                width="120"
+                src={stringHelper('LOGO_URL')}
+              />
+            )}
+          </FormattedGlobalMessage>
+          <StyledSubHeader className="reset-password__heading">
+            { showConfirmDialog ?
+              <FormattedMessage id="passwordChange.successTitle" defaultMessage="Password updated" />
+              : <FormattedMessage id="passwordChange.title" defaultMessage="Change password" />
+            }
+          </StyledSubHeader>
 
-                    <ChangePasswordComponent
-                      type="reset-password"
-                      show_current_password={false}
-                      token={token}
-                      show_confirm={this.showConfirm.bind(this)}
-                    />
-                  </CardContent>
-                </Card>
-              }
-            </StyledPasswordChange>
-          </PageTitle>
-        )}
-      </FormattedMessage>
-    );
-  }
+          { showConfirmDialog ?
+            <div>
```

Comment:
Grouping divs can be replaced with `React.Fragment`

Reply:
Done